stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -397,6 +397,8 @@ function SparseMatrixCSC{Tv,Ti}(M::StridedMatrix) where {Tv,Ti}
     end
     return SparseMatrixCSC(size(M, 1), size(M, 2), colptr, rowval, nzval)
 end
+SparseMatrixCSC(M::LinearAlgebra.Adjoint{Tv,SparseMatrixCSC{Tv,Ti}}) where {Tv,Ti} = copy(M)
```

Comment:
Should probably relax the signature to `Adjoint{<:Any,<:SparseMatrixCSC}` (which is also the one of the `copy` methods this should dispatch to); likewise below.

Reply:
Hmm

```
julia> B = sprand(1,1,1.0);

julia> SparseMatrixCSC(M::Adjoint{<:Any,SparseMatrixCSC}) = copy(M)
SparseMatrixCSC

julia> @which SparseMatrixCSC(B')
(::Type{SparseMatrixCSC})(M::AbstractArray{Tv,2}) where Tv in SparseArrays at /Users/kristoffer/julia/usr/share/julia/stdlib/v0.7/SparseArrays/src/sparsematrix.jl:371

julia> SparseMatrixCSC(M::LinearAlgebra.Adjoint{Tv,SparseMatrixCSC{Tv,Ti}}) where {Tv,Ti} = copy(M)
SparseMatrixCSC

julia> @which SparseMatrixCSC(B')
(::Type{SparseMatrixCSC})(M::Adjoint{Tv,SparseMatrixCSC{Tv,Ti}}) where {Tv, Ti} in Main at REPL[7]:1
```

Reply:
`Adjoint{<:Any,<:SparseMatrixCSC}`, not `Adjoint{<:Any,SparseMatrixCSC}`:
```julia
julia> B = sprand(1,1,1.0);

julia> SparseMatrixCSC(M::Adjoint{<:Any,<:SparseMatrixCSC}) = copy(M)
SparseMatrixCSC

julia> @which SparseMatrixCSC(B')
(::Type{SparseMatrixCSC})(M::Adjoint{#s2,#s1} where #s1<:SparseMatrixCSC where #s2) in Main at REPL[19]:1
```

Reply:
Oops, sorry.

Reply:
Fixed.